src/test/java/Homework21.java

```diff
@@ -25,7 +27,7 @@ public void playSong()   {
     //      renames playlist using Actions double click (Pre-requisite - create at least one playlist)
     @Test
     public void renamePlaylist() throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

![image](https://user-images.githubusercontent.com/104761482/213122660-a116892b-b8f2-4b8d-9cbd-04445d23b7f8.png)